### ISSUE TITLE
Bittrex bulk cancel (by Currency Pair) is not implemented 

### ISSUE THREAD

**Issue by appyhour770** [knowm/XChange#3734]
Bittrex bulk cancel (by CancelOrderByCurrencyPair ) does not seem to be implemented. If possible, please also provide example of use

**Comment by cymp** [comment]
Easy to implement. From https://bittrex.github.io/api/v3:

09/23/2020
You now cancel all of your open orders or all of your open orders on a market with DELETE /orders/open

**Comment by appyhour770** [comment]
Yes that's what my bot is doing already, but with 200 orders, it takes some (precious) time.
That is the interest of having 1 call for 1 cancelALL (more rapid in some situations)

**Comment by cymp** [comment]
I don't think you read well.

**Comment by appyhour770** [comment]
Sorry... You are right, it'es easy to implement (and yet to be implemented)

**Comment by cymp** [comment]
I can suggest you to offer a bounty if you want this feature to be available quick.
https://github.com/knowm/XChange/issues?q=label%3ABounty+
